### PR TITLE
Migrate watcher hlrc response tests to use AbstractResponseTestCase

### PR DESCRIPTION
**Pull request by martijnvg**
Relates to #43472

### CODE REVIEW

File: client/rest-high-level/src/test/java/org/elasticsearch/client/watcher/ExecuteWatchResponseTests.java [hunk 1]
```diff
@@ -1,115 +0,0 @@
...
-
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.List;
-import java.util.Map;
-
-import static org.elasticsearch.test.AbstractXContentTestCase.xContentTester;
-import static org.hamcrest.Matchers.is;
-
-public class ExecuteWatchResponseTests extends ESTestCase {
```
Comment (martijnvg) [inline client/rest-high-level/src/test/java/org/elasticsearch/client/watcher/ExecuteWatchResponseTests.java]: Duplicate of `ExecuteWatchResponseTests`

**Review by hub-cap — APPROVED** [review]
mucho better ty for doing this!

**Comment by martijnvg** [comment]
@elasticmachine update branch

**Comment by martijnvg** [comment]
@elasticmachine run elasticsearch-ci/2

**Pull request merged**